### ISSUE TITLE
Markdown cell content not read out using VoiceOver  and new markdown renderer

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#118594]
**Repro**
1. With `"notebook.experimental.useMarkdownRenderer": true`
2. Open a notebook with rendered markdown cells
3. Enable VoiceOver 
4. Try using arrow keys to navigate through the markdown cells

**Bug**
With the old markdown renderer this sort of worked (it seems to only read the first line). With the new renderer, it just reads: `group` for all markdown cells

**Comment by mjbvz** [comment]
Quick review of what I've tried so far to fix this:

## Use aria attributes in the workbench dom

This involved setting  `aria-label` on the main cell node in the notebook dom.

This causes Voiceover to read out the markdown text, however I ran into a few problems/limitations:

* The label needs to use the rendered markdown text, not the markdown markup. 
* This does not appear to support many of the screenreader navigation commends, such as navigating between paragraphs for long markdown content (not 100% sure I was testing this correctly and maybe there's another attribute we can use instead?)


## Focus into the webview when a markdown cell is focused

This calls `.focus` on the rendered markdown cell inside the webview.

This also works but needs major work to polish:

* focusing into the webview causes the cell to lose its active focus border
* Once focused inside the webview, the tab order seems to be very odd (perhaps this has already been solved for output views?)
* Voiceover will sometimes mention that you are inside a frame and offer frame navigation options. This is an implementation detail and should likely not be exposed to users

/cc @rebornix

**Comment by mjbvz** [comment]
For now, we've disabled the new markdown renderers when in screen reader mode. Will continue investigating proper support  accessibility for the new markdown renderer

**Comment by mjbvz** [comment]
This should have been fixed by 27858a0faf78e502acf3c8f620f86a671a55e473

Let me know if you are still seeing any issues
